portal-frontend/src/app/services/notice-of-intent-submission/notice-of-intent-submission.dto.ts

```diff
@@ -146,4 +146,5 @@ export interface NoticeOfIntentSubmissionUpdateDto {
 export interface ProposedStructure {
   type: STRUCTURE_TYPES | null;
   area: number | null;
+  options?: any;
```

Comment:
Can you please give details on using 'any' type since it would bypass ts type checking and can cause runtime errors in the future? Are we doing validation?

Reply:
Handled !